stacks/lemp/service-config/nginx.conf

```diff
@@ -0,0 +1,82 @@
+worker_processes 4;
+pid /var/run/nginx.pid;
+
+events {
+	worker_connections 768;
+	# multi_accept on;
+}
+
+http {
+	# Basic Settings
+	sendfile on;
```

Comment:
See "Known Issues with sendfile(2)" on http://www.proftpd.org/docs/howto/Sendfile.html

Maybe I've spent too many years administering NFS and AFS.